packages/plugin/src/generators/e2e-project/files/tests/__pluginName__.spec.ts__tmpl__

```diff
@@ -1,37 +1,37 @@
-import {
-  checkFilesExist,
-  ensureNxProject,
-  readJson,
-  runNxCommandAsync,
-  runNxCommand,
-  uniq,
-} from '@nx/plugin/testing';
+import { ensureNxProject } from '@nx/plugin/testing';
+import { existsSync } from 'fs';
+import { join } from 'path';
+import { execSync } from 'child_process';
 
 describe('<%= pluginName %> e2e', () => {
   // Setting up individual workspaces per
   // test can cause e2e runs to take a long time.
-  // For this reason, we recommend each suite only
-  // consumes 1 workspace. The tests should each operate
-  // on a unique project in the workspace, such that they
-  // are not dependant on one another.
+  // For this reason, we recommend setting up the workspace in a beforeAll.
+  let workspaceDirectory: string;
   beforeAll(() => {
-    ensureNxProject('<%= npmPackageName %>', '<%= pluginOutputPath %>');
+    // Setup a test workspace on disk
+    workspaceDirectory = ensureNxProject('<%= npmPackageName %>', '<%= pluginOutputPath %>');
   });
 
-  afterAll(() => {
+  afterAll(async () => {
     // `nx reset` kills the daemon, and performs
     // some work which can help clean up e2e leftovers
-    runNxCommandAsync('reset');
+    // await unlink(cwd);
```

Comment:
This will fail if daemon is active, since there is an open file handle to the log file